internal/storage/types/ssblob/ssblob.go

```diff
@@ -0,0 +1,135 @@
+// Package ssblob provides a blob implementation for the Archivematica Storage
+// Service. Use OpenBucket to construct a *blob.Bucket.
+package ssblob
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/url"
+
+	"gocloud.dev/blob"
+	"gocloud.dev/blob/driver"
+	"gocloud.dev/gcerrors"
+)
+
+type NotImplemented error
+
+var errNotImpl NotImplemented
```

Comment:
I think that you should go back to the solution I suggested in my [draft package](https://github.com/sevein/ssblob/blob/9a6d4024edbc25298ad61b1dcc2e7ad79efb14a8/ssblob.go#L18):

```go
var errNotImplemented = errors.New("not implemented")
```

We don't need to define a custom error type or export it, I believe that'd be unnecessary complexity. We're only expected to export gcerrors using the `ErrorCode` method that is part of the interface, like this:

```go
var (
       errNotFound       = errors.New("blob not found")
       errNotImplemented = errors.New("not implemented")
)

func (b *bucket) ErrorCode(err error) gcerrors.ErrorCode {
       switch err {
       case errNotFound:
               return gcerrors.NotFound
       case errNotImplemented:
               return gcerrors.Unimplemented
       default:
               return gcerrors.Unknown
       }
}
```

For example: https://github.com/google/go-cloud/blob/4e3dfbec38511935040c9805491cb8ac1341b799/blob/fileblob/fileblob.go#L340-L347.